README.md

```diff
@@ -266,6 +266,37 @@ print(result.response)
 
 </details>
 
+
+<details>
+  <summary>Use LeMUR to with Input Text</summary>
```

Comment:
nit: Is there a typo here? "Use LeMUR to with"?

Reply:
Ah yes. I think we should follow up on that in another change. The feedback loop is very long on this. Multiple PRs and 1+ hour of tests to run.